commit a95eeb9c8bc3d3b48ab5a2b9306d83cac3b9b2f4
Author: Olly Betts <olly@survex.com>
Date:   Mon Dec 2 12:31:31 2024 +1300

    Adjust lollipop testcase to pass on i386
    
    Excess precision was changing which station was picked as the
    southernmost.

diff --git a/tests/lollipop.out b/tests/lollipop.out
index d17aece8..295352ab 100644
--- a/tests/lollipop.out
+++ b/tests/lollipop.out
@@ -13,14 +13,14 @@ Calculating trailing traverses...
 
 Calculating statistics...
 
-Survey contains 5 survey stations, joined by 5 legs.
+Survey contains 6 survey stations, joined by 6 legs.
 There is 1 loop.
-Total length of survey legs =   13.00m (  11.00m adjusted)
-Total plan length of survey legs =   13.00m
+Total length of survey legs =   14.00m (  12.00m adjusted)
+Total plan length of survey legs =   14.00m
 Total vertical length of survey legs =    0.00m
-Vertical range = 0.00m (from c at 0.00m to c at 0.00m)
-North-South range = 5.00m (from c at 5.00m to 1 at 0.00m)
+Vertical range = 0.00m (from s at 0.00m to s at 0.00m)
+North-South range = 6.00m (from c at 5.00m to s at -1.00m)
 East-West range = 4.00m (from 2 at 4.00m to 1 at 0.00m)
    2 1-nodes.
-   1 2-node.
+   2 2-nodes.
    2 3-nodes.
diff --git a/tests/lollipop.pos b/tests/lollipop.pos
index ac6eb12f..9abb42e2 100644
--- a/tests/lollipop.pos
+++ b/tests/lollipop.pos
@@ -1,6 +1,7 @@
 ( Easting, Northing, Altitude )
 (    0.00,     0.00,     0.00 ) 1
 (    4.00,     0.00,     0.00 ) 2
-(    2.00,    -0.00,     0.00 ) a
-(    2.00,     3.00,     0.00 ) b
-(    2.00,     5.00,     0.00 ) c
+(    2.00,     0.00,     0.00 ) a
+(    2.00,     3.00,    -0.00 ) b
+(    2.00,     5.00,    -0.00 ) c
+(    0.00,    -1.00,     0.00 ) s
diff --git a/tests/lollipop.svx b/tests/lollipop.svx
index 9b687686..350a6d8c 100644
--- a/tests/lollipop.svx
+++ b/tests/lollipop.svx
@@ -6,3 +6,4 @@ A 2 3 090 0
 A B 3 000 0
 B C 2 000 0
 C B 2 180 0
+1 S 1 180 0 ; Create stable south-most station
